swift/Samples/TabBarContainer/DemoApp/Bar/BarScreen.swift

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2019 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import Workflow
+import WorkflowUI
+
+
+struct BarScreen: Screen {
```

Comment:
I’d maybe avoid “Bar” as the name here, since it’s easy to confuse with the bar from “TabBar”

Reply:
Will rename to my next favorite name "Baz".